### DISCUSSION TITLE
来交换涩图吧

### DISCUSSION THREAD

**Discussion by Anai136** [share121/inter-knot#5469] [灌水]
![Uploading 1723848557534424ed6a84086c5c27a8e4b3d9c9ebccb9d3db3d03b34c44337b6175b0c49c5b6.0.jpg…]()

**Comment by Anai136** [comment]
![Uploading 17241537311070effd7193dca1bfbfc2ab7dd24f345547527fc3033a52ef2c983d9244077d9fa.0.jpg…]()

**Reply by BJ-cmyk** [reply]
孩子，咱上传图片的时候要等一小会儿的
